quickstarts/microsoft.web/web-app-asp-app-on-asev3-create/modules/site.bicep

```diff
@@ -0,0 +1,87 @@
+@description('Required. App service prefix.')
+param appNamePrefix string = 'app-2'
+
+@description('Required. App service location.')
+param location string = resourceGroup().location
+
+@description('Required. App service plan prefix.')
+param hostingPlanNamePrefix string = 'asev3-asp-2'
+
+@description('Required. App service plan hosting environment profile name (ASEv3 name).')
+param hostingEnvironmentProfileName string = 'asev3-ilb-cua3tpryw2j64'
+
+@description('Required. Enable Always-on of App service.')
```

Comment:
remove "required" from the description for all parameters (all params are required)

Reply:
done.